sdk/storage/Azure.Storage.Common/src/Azure.Storage.Common.csproj

```diff
@@ -18,7 +18,8 @@
     <RootNamespace>Azure.Storage</RootNamespace>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Azure.Core" />
+    <!-- TODO (kasobol-msft) undo before merge when core ships. -->
```

Comment:
TODO undo this.

Reply:
Might switch to `<UseProjectReferenceToAzureClients>true</UseProjectReferenceToAzureClients>` to make it mergable and switch it back before release.

Reply:
@seanmcc-msft FYI. We have to remember this when we do release.